src/test/java/seedu/address/model/person/DebtTest.java

```diff
@@ -15,9 +15,10 @@ public void isValidDebt() {
         assertFalse(Debt.isValidDebt("phone")); // non-numeric
         assertFalse(Debt.isValidDebt("9011p041")); // alphabets within digits
         assertFalse(Debt.isValidDebt("9312 1534")); // spaces within digits
-
-        // valid phone numbers
+        assertFalse(Debt.isValidDebt("123.345")); // more than two decimal places
+        // valid debts
```

Comment:
Hey jel, just a query, should we comment in this way: "//.........//" or this way: "/(asterisk)...........(asterisk)/"

Reply:
`//` for single-line comments. `/**/` for multiple-lines comments